types/documentdb/index.d.ts

```diff
@@ -126,12 +129,8 @@ interface AbstractMeta extends UniqueId {
 
 /** Represents a custom document for storing in DocumentDB  */
 interface NewDocument<TContent> extends UniqueId {
```

Comment:
Since `doc` has been removed `NewDocument` is no longer generic. Either add `doc` back or remove the generic parameter.